### ISSUE TITLE
wrong version number in macOS download 0.3.4.223

### ISSUE THREAD

**Issue by suschizu** [contour-terminal/contour#841]
### Contour Terminal version

0.3.4.223

### Installer source

GitHub: release page

### Operating System

OS/X Monterey 12.6

### Architecture

x86-64

### Steps to reproduce

the info.plist contains a wrong version notation: 

```
CFBundleIdentifier:
org.contour-terminal.contour

CFBundleShortVersionString:
0.1.0_pre

CFBundleVersion:
0.1.0_pre
```

<img width="376" alt="Screenshot 2022-10-03 at 17 28 33" src="https://user-images.githubusercontent.com/16004333/193617402-b2017acc-b90a-448b-908e-7b3612e60f76.png">

### Expected Behavior

CFBundleShortVersionString (and eventually CFBundleVersion) should contain 0.3.4.223

### Actual Behavior

shows a prerelease version: 0.1.0_pre

**Comment by theoniko** [comment]
Hello @christianparpart,
As far as i understood, i need to correct  the contour_version of Cmakelists (set(contour_version "0.1.0_pre" line) which then is used by MacOSXBundleInfo.plist.in. Is my understaning correct? If not, could you please give any hint for this issue?

**Comment by uspasojevic96** [comment]
@theoniko That is correct

**Comment by christianparpart** [comment]
Oh nice. Many thanks for your interest in helping me out here. :)

**Comment by theoniko** [comment]
Please refer to #871. Values will as follows
<img width="477" alt="Versions" src="https://user-images.githubusercontent.com/28800819/200180753-902d716e-3634-4e71-920f-1e08515e3fd1.png">

**Comment by christianparpart** [comment]
> Please refer to #871. Values will as follows <img alt="Versions" width="477" src="https://user-images.githubusercontent.com/28800819/200180753-902d716e-3634-4e71-920f-1e08515e3fd1.png"> I hope everything is correct.

Sounds about right, I think Github CI would also complain otherwise. 😀

**Comment by christianparpart** [comment]
This was fixed in #871. Thanks again, for that. :)
